swov1/.speakeasy/tests.arazzo.yaml

```diff
@@ -1,1285 +1,157 @@
-                "lastTestTime": "2025-01-15T14:31:19.735Z",
-                "lastErrorTime": "2025-01-15T14:31:19.735Z",
-                "lastResponseTime": 376
-              }
-            type: simple
-    x-speakeasy-test-group: dem
-  - workflowId: updateUri
-    steps:
-      - stepId: test
-        operationId: updateUri
-        parameters:
-          - name: entityId
-            in: path
-            value: <id>
-        requestBody:
-          contentType: application/json
-          payload:
-            availabilityCheckSettings:
-              outageConfiguration:
-                consecutiveForDown: 2
-                failingTestLocations: all
-              platformOptions:
-                probePlatforms:
-                  - AWS
-                testFromAll: true
-              tcp:
-                enabled: true
-                port: 443
-              testFrom:
-                type: REGION
-                values:
-                  - NA
-              testIntervalInSeconds: 300
-            ipOrDomain: solarwinds.com
-            name: example-uri
-            tags:
-              - key: environment
-                value: production
-        successCriteria:
-          - condition: $statusCode == 200
-          - condition: $response.header.Content-Type == application/json
-          - context: $response.body
-            condition: |
-              {
-                "id": "e-1448474379026206720"
-              }
-            type: simple
-    x-speakeasy-test-group: dem
-  - workflowId: deleteUri
-    steps:
-      - stepId: test
-        operationId: deleteUri
-        parameters:
-          - name: entityId
-            in: path
-            value: <id>
-        successCriteria:
-          - condition: $statusCode == 200
-          - condition: $response.header.Content-Type == application/json
-          - context: $response.body
-            condition: |
-              {
-                "id": "e-1448474379026206720"
-              }
-            type: simple
-    x-speakeasy-test-group: dem
-  - workflowId: pauseUriMonitoring
-    steps:
-      - stepId: test
-        operationId: pauseUriMonitoring
-        parameters:
-          - name: entityId
-            in: path
-            value: <id>
-        successCriteria:
-          - condition: $statusCode == 200
-          - condition: $response.header.Content-Type == application/json
-          - context: $response.body
-            condition: |
-              {
-                "id": "e-1448474379026206720"
-              }
-            type: simple
-    x-speakeasy-test-group: dem
-  - workflowId: unpauseUriMonitoring
-    steps:
-      - stepId: test
-        operationId: unpauseUriMonitoring
-        parameters:
-          - name: entityId
-            in: path
-            value: <id>
+            value: "2025-06-15T00:00:00Z"
+          - name: endTime
+            in: query
+            value: "2025-07-22T23:59:59Z"
```

Comment:
Eventually these will return no data as logs age out. Can we talk to speakeasy about being able to generate these somehow?